src/Sylius/Behat/Context/Ui/Shop/ProductContext.php

```diff
@@ -203,13 +204,21 @@ public function theLastAttributeShouldBe($name)
     }
 
     /**
-     * @When /^I browse products from (taxon "([^"]+)")$/
+     * @When I browse products from taxon :taxon
      */
     public function iCheckListOfProductsForTaxon(TaxonInterface $taxon)
     {
         $this->indexPage->open(['slug' => $taxon->getSlug()]);
     }
 
+    /**
+     * @When I try to browse products from taxon :taxon with a trailing slash in the path
+     */
+    public function iTryToBrowseProductsFromTaxonWithATrailingSlashInThePath(TaxonInterface $taxon)
```

Comment:
Missing `: void` here and in the second added function

Reply:
@Zales0123 I've just fixed a lot of missing void in this file.